contracts/partners/giveth/GivethMetadata.sol

```diff
@@ -0,0 +1,65 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+pragma solidity ^0.8.4;
+
+import "base64-sol/base64.sol";
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/Strings.sol";
+import "../../lib/strings.sol";
+
+/// @title Domain metadata contract
+/// @author Tempe Techie
+/// @notice Contract that stores metadata for a TLD
+contract GivethMetadata is Ownable {
+  // @todo
+  string public description = "Giveth Domains";
+  string public brand = "";
```

Comment:
@chnejohnson Brand should be called "Giveth Names".

For description please use "Giveth Names are web3 usernames brought to you by Giveth and Punk Domains. Proceeds from minting new .giveth names go into the Giveth Mathcing Pool."